### ISSUE TITLE
Vera++ configuration file missing from repository

### ISSUE THREAD

**Issue by heplesser** [nest/nest-simulator#445]
According to [NEST Developer Space](https://nest.github.io/nest-simulator/coding_guidelines_c++#vera-profile-nest), the NEST repository should contain a configuration file for vera++, that needs to be symlinked from the vera++ profiles directory. This file is currently missing. On Travis, `build.sh` generates that file on the fly with the following content:

```
#!/usr/bin/tclsh
# This profile includes all the rules for checking NEST
set rules {
  F001
  F002
  L001
  L002
  L003
  L005
  L006
  T001
  T002
  T004
  T005
  T006
  T007
  T010
  T011
  T012
  T013
  T017
  T018
  T019
}
```

This is not an option for `check_code_style.sh`, since we cannot be sure to have write permission to the vera++ profiles directory. So we need to include the file somewhere. `build.sh` should then use that file instead of generating one on the fly to ensure consistency.

**Comment by heplesser** [comment]
@jougs Could you take a look at this one?

**Comment by heplesser** [comment]
Fixed by merge of #562.
